azure-pipelines.yml

```diff
@@ -317,18 +317,9 @@ stages:
             inputs:
               artifactName: Envoy.framework
               targetPath: dist/Envoy.framework
-          - script: ./bazelw build --config=ios //examples/swift/hello_world:app
-            displayName: 'Build swift app'
-          # Now check that the app actually runs on the simulator.
-          # This is a non-ideal way to check for liveliness, but works for now.
-          # First start the iOS simulator.
-          # Interestingly bazel run does not start the simulator in CI.
-          # https://github.com/lyft/envoy-mobile/issues/201 for further investigation.
-          - script: npm install -g ios-sim && ios-sim start --devicetypeid "iPhone-X, 12.2"
-            displayName: 'Start the iOS simulator'
           # Run the app in the background and redirect logs.
           - script: ./bazelw run --config=ios //examples/swift/hello_world:app &> /tmp/envoy.log &
```

Comment:
@keith I am not convinced this is starting the simulator (per the details I put in https://github.com/lyft/envoy-mobile/issues/201). I am almost positive that if you take a look at the redirected logs you will see what I posted in the linked issue.

Reply:
I think I'll wait until this test is re-enabled to try and fix this

Reply:
Totally fair. Do you mind if I close this PR for the time being?